sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/RegistryArtifact.java

```diff
@@ -295,7 +295,7 @@ public ArtifactTagProperties getTagProperties(String tag) {
      *
      * <!-- src_embed com.azure.containers.containerregistry.RegistryArtifact.listTagPropertiesWithOptions -->
      * <pre>
-     * client.listTagProperties&#40;ArtifactTagOrderBy.LAST_UPDATED_ON_DESCENDING, Context.NONE&#41;
+     * client.listTagProperties&#40;ArtifactTagOrder.LAST_UPDATED_ON_DESCENDING, Context.NONE&#41;
```

Comment:
This codesnippet looks to be pointing to the wrong API for the method it is being used as documentation. This API doesn't take any parameters.

Reply:
Good catch. Thanks!